src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,217 @@
+import React, { useState, useEffect, useRef } from "react";
+import "./Navbar.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { Button, Dropdown, DropdownButton } from "react-bootstrap";
+import {
+  List,
+  HouseDoor,
+  Search,
+  Gift,
+  Heart,
+  InfoCircle,
+  Envelope,
+  GeoAlt,
+} from "react-bootstrap-icons";
+import { Link, BrowserRouter as Router } from "react-router-dom";
+
+const NavbarForLargeScreens = () => {
+  return (
+    <Router>
+      <div id="navbar">
+        <div id="navbarLogo">
+          <Link to="/">LOGO</Link>
+        </div>
+        <div id="navbarLinks">
+          <Link to="/" className="eachLinksInNavbar">
```

Comment:
I see some repetetion here form line 25 till line 45 this can be done using an array of objects and then mapping over it to create each ```Link```